frontend/components/campaigns/dashboard/CampaignInfoCard.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { CampaignBaseData } from '../../../types/campaigns';
+import { Grid, Tooltip, Typography } from '@mui/material';
+import { Box, Stack } from '@mui/system';
+import CalendarTodayIcon from '@mui/icons-material/CalendarToday';
+import { DATE_FORMAT } from '../../../utils/constants';
+import {
+  campaignInfoItemSx,
+  campaignDateIconSx,
+  campaignImageSx,
+  campaignMoneyIconSx,
+} from '../../../styles/components/campaigns/dashboard/CampaignDashboardStyles';
+import LinearScaleIcon from '@mui/icons-material/LinearScale';
+import moment, { Moment } from 'moment';
+import PaidIcon from '@mui/icons-material/Paid';
+import LocalActivityIcon from '@mui/icons-material/LocalActivity';
+
+interface Props {
+  campaignBaseInfo: CampaignBaseData;
+}
+
+interface CampaignMoneyInfoProps {
+  tooltipTitle: string;
+  icon: React.ReactNode;
+  amount: number;
+}
+
+interface CampaignDateInfoProps {
+  date: Moment;
+}
+
+const CampaignMoneyInfoIcon = ({ tooltipTitle, icon, amount }: CampaignMoneyInfoProps) => (
+  <Tooltip title={tooltipTitle}>
+    <Stack sx={campaignMoneyIconSx} direction="row" component="div" spacing={1}>
+      {icon}
+
+      <Typography variant="h4">{amount}</Typography>
+    </Stack>
+  </Tooltip>
+);
+
+const CampaignDateInfoIcon = ({ date }: CampaignDateInfoProps) => (
+  <Stack sx={campaignDateIconSx}>
+    <CalendarTodayIcon fontSize="large" />
+
+    <Typography>{date.format(DATE_FORMAT)}</Typography>
+  </Stack>
+);
+
+const CampaignInfoCard = ({ campaignBaseInfo }: Props) => {
+  const getStatus = (start: Moment, end: Moment) => {
+    if (end.isBefore(moment().startOf('day'))) {
```

Comment:
Btw @sivayogasubramanian is this same logic using `startOf('day')` and `endOf('day')` supposed to apply to the campaigns list too? Because otherwise it shows the wrong status if the campaign starts and end on the same day?